tests/parm/nems.configure.atmw.IN

```diff
@@ -68,9 +68,6 @@ DRIVER_attributes::
 MED_attributes::
       ATM_model = @[atm_model]
       WAV_model = @[wav_model]
-      history_n = 1
```

Comment:
If we remove this line, what default value history_n has?

Reply:
This is an optional configuration that is behind an if present, if set logical. So it usually defaults to -999 if not set.

This way of specifying the writing of optional history files has been super-ceded by the ability to write component history files. The older style requires the addition of a mediator history write phase in the run sequence and all components write into the same file. These settings do nothing w/o that history write phase being present in the run sequence.

The newer functionality does not require any change to the run sequence---mediator history files can be turned on/off with configuration settings and each component can output multiple types of history files---instantaneous, time-averaged etc.

Reply:
@junwang-noaa, would you consider Denise's answer to be a resolution to this discussion / review?

Reply:
I believe this conversation is resolved, continuing with merge process.